doc/chapters/integration-guidelines/10-systemd-service.rst

```diff
@@ -0,0 +1,22 @@
+systemd.service
+===============
+
+As it is mentioned in :ref:`Getting Started <getting-started>` section the user API is implemented by
+the SoftwareContainer Agent as a D-bus service. The API is used to start, configure, and control
+containers. Thus in order to use SoftwareContainers the agent should be run first. To be able to start
```

Comment:
I would phrase this as "SoftwareContainer containers" instead of "SoftwareContainers"